### PR TITLE
Chapter 8: Basic login

### PR DESCRIPTION
**Pull request by thanhtv-2822**
- login, logout, set session
Screenshot from 2021-11-30 18-35-02

### CODE REVIEW

**Comment by thanhtv-2822** [comment]
Ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
+          </li>
+          <li>
+            <%= link_to t("nav.home") , home_path %>
+          </li>
+          <li>
+            <%= link_to t("nav.help") , help_path %>
+          </li>
+          <% if logged_in? %>
+            <li>
+              <%= link_to t(".users") , '#' %>
```
Comment (anlvq-1058) [inline app/views/layouts/_header.html.erb:20]: Dùng "

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,27 +1,48 @@
...
+                <li>
+                  <%= link_to t("nav.logout") , logout_path, method: :delete %>
+                </li>
+              </ul>
+            </li>
+            <% else %>
+              <li>
+                <%= link_to t("nav.login") , login_path %>
+              </li>
+              <% end %>
```
Comment (anlvq-1058) [inline app/views/layouts/_header.html.erb:44]: indent

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<% provide :title, t("nav.login") %>
+<h1 class="log-in">Log in</h1>
```
Comment (anlvq-1058) [inline app/views/sessions/new.html.erb:2]: i18n

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Pull request merged**